docs/baseband/dada/index.rst

```diff
@@ -37,29 +42,17 @@ are loaded into memory (since the sample file is encoded using 8 bits, the
 above example thus loads 12 bytes into memory).
 
 Opening in stream mode wraps the low-level routines such that reading and
-writing is in units of samples, and one has access to header information.
+writing is in units of samples, and provides access to header information.
+
+To set up a file for writing as a stream is possible as well.  Here, we use an
+even smaller size of the payload, to show how one can define multiple files.
 
 ::
 
     >>> fh = dada.open(SAMPLE_DADA, 'rs')
-    >>> fh
```

Comment:
My thinking was to remove all examples that are already in Getting Started, to minimize duplication.  I'm on the fence about it, because it's still useful to have a short, self-contained tutorial for each format.  I think that whatever we do here we must also do for VDIF, though.

Reply:
On further consideration, I've restored it.